From: SeongJae Park <sj@kernel.org>
Date: Fri, 7 Feb 2025 13:07:37 -0800
Subject: [PATCH] ==== large folios fix ====

mm/damon/paddr: fix large folios access and schemes handling

DAMON operations set for physical address space, namely 'paddr', treats
tail pages as unaccessed always.  It can also apply DAMOS action to
a large folio multiple times within single DAMOS' regions walking.  As a
result, the monitoring output has poor quality and DAMOS works in
unexpected ways when large folios are being used.  Fix those.

The patches were parts of Usama's hugepage_size DAMOS filter patch
series[1].  The first fix has collected from there with a slight commit
message change for the subject prefix.  The second fix is re-written by
SJ and posted as an RFC before this series.  The second one also got a
slight commit message change for the subject prefix.

[1] https://lore.kernel.org/20250203225604.44742-1-usamaarif642@gmail.com
[2] https://lore.kernel.org/20250206231103.38298-1-sj@kernel.org

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 hkml_cv_bogus/hkml_cv_bogus_bfb91yub | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 hkml_cv_bogus/hkml_cv_bogus_bfb91yub

diff --git a/hkml_cv_bogus/hkml_cv_bogus_bfb91yub b/hkml_cv_bogus/hkml_cv_bogus_bfb91yub
new file mode 100644
index 000000000000..e69de29bb2d1
-- 
2.39.5

